### PR TITLE
Fix Issue 19775 - multiple argument string mixin doesn't expand tuples

### PR DESCRIPTION
**Pull request by ntrel**

### CODE REVIEW

**Review by thewilsonator — APPROVED** [review]

**Comment by thewilsonator** [comment]
```
compilable/b19775.d-mixin-3(3): Error: function expected before `()`, not `42` of type `int`
compilable/b19775.d(5): Error: template instance `b19775.ident!"x1"` error instantiating
compilable/b19775.d(10):        while evaluating: `static assert(x2 == 42)`
compilable/b19775.d(11):        while evaluating: `static assert(x3 == 42)`
compilable/b19775.d(12):        while evaluating: `static assert(x4 == 42)`
compilable/b19775.d(13):        while evaluating: `static assert(x5 == 42)`
```

File: test/compilable/b19775.d [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// https://issues.dlang.org/show_bug.cgi?id=19775
+enum x1 = 42;
+enum ident(alias a, args...) = mixin(a, args);
+
+//enum x2 = ident!("x1"); //FIXME - empty args
+enum x2 = x1;
```
Comment (ntrel) [inline test/compilable/b19775.d]: For now I've given up on getting an empty sequence to be ignored by `mixin()`. It still gets turned into `"()"`.

**Review by PetarKirov — APPROVED** [review]

**Comment by ntrel** [comment]
@thewilsonator I fixed the test failures, except I don't know why semaphoreci is failing.

**Comment by thewilsonator** [comment]
Semaphore occasionally fails with reproducibility test, dunno why.

**Pull request merged**